tailwind.config.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
-  mode: 'jit',
-  purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
-  darkMode: false,
+  content: [
+    './pages/**/*.{js,ts,jsx,tsx}',
+    './components/**/*.{js,ts,jsx,tsx}',
+  ],
   theme: {
+    colors: {
```

Comment:
Do we want to completely replace all of the tailwind colours? I didn't think we did

Reply:
Good catch, didn't realize I hadn't put this inside `extend`, which also explains the issue I was having with `ring-black` in `globals.css`

Reply:
you fixed this, @krischarbonneau ?

Reply:
yes, this was resolved with my latest commit @P0NDER0SA